src/elements/content-preview/preview-header/PreviewHeaderComponent.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @flow
+ * @file Preview header container component
+ * @author Box
+ */
+
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames';
+import Header from './Header';
+import VersionsHeader from './VersionsHeader';
+import './Header.scss';
+
+type Props = {
+    canAnnotate: boolean,
+    canDownload: boolean,
+    contentOpenWithProps?: ContentOpenWithProps,
+    file?: BoxItem,
+    isPreviewingCurrentVersion: boolean,
+    onClose?: Function,
+    onDownload: Function,
+    onPrint: Function,
+    token: ?string,
+} & InjectIntlProvidedProps;
+
+const PreviewHeaderComponent = ({
+    canAnnotate,
+    canDownload,
+    contentOpenWithProps,
+    file,
+    isPreviewingCurrentVersion,
```

Comment:
Can be derived by passing in `version`, which will also be necessary to display the correct file name and type.